### PR TITLE
Add helm

### PR DESCRIPTION
**Pull request by zhangzerui20**
#161 

- [ ] Configuration
- [ ] Docs
- [x] Installation
- [ ] Performance and Scalability
- [ ] Naming
- [ ] HealthCheck
- [ ] Test and Release

**Please check any characteristics that apply to this pull request.**

- [ ] Does not have any user-facing changes. This may include API changes, behavior changes, performance improvements, etc.

### CODE REVIEW

File: deploy/helm/values.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  image:
+    repository: polarismesh/polaris-server
+    tag: latest
+    pullPolicy: Always
+  limit:
+    cpu: "500m"
+    memory: "1000Mi"
+  console:
+    image:
+      repository: ccr.ccs.tencentyun.com/polaris_mesh/polaris-console
```
Comment (chuntaojun) [inline deploy/helm/values.yaml:19]: 这里在dockerhub上已经发布了polaris-console，这里应该可以改过下

File: deploy/helm/README.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+Uninstall your installed release with the command below, replacing `${release_name}` with the release name you used.
+
+```shell
+$ helm uninstall `${release_name}`
+```
+
+## Configuration
+The currently supported configurations are as follows:
+
+| 参数名                                | 参数解释                              |
```
Comment (chuntaojun) [inline deploy/helm/README.md:70]: 这里需要专为英文下

File: deploy/helm/templates/polaris-console-config.yaml [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+      listenIP: "0.0.0.0"
+      listenPort: {{ .Values.service.webPort }}
+      requestURL: "/naming/v1"
+      monitorURL: "/api/v1"
+      webPath: "web/dist/"
+    polarisServer:
+      address: "127.0.0.1:{{ .Values.service.httpPort }}"
+      polarisToken: "polaris@12345678"
+    monitorServer:
+      address: "polaris-prometheus:9090"
```
Comment (chuntaojun) [inline deploy/helm/templates/polaris-console-config.yaml:24]: 我觉得这里也可以改为可配置的吧？用户直接自己接入自己的prometheus服务

File: deploy/helm/values.yaml (hunk 1 above)
Comment (zhangzerui20) [reply deploy/helm/values.yaml]: 修复了。polaris-server 也换成最新的 tag 了。

File: deploy/helm/README.md (hunk 2 above)
Comment (zhangzerui20) [reply deploy/helm/README.md]: 改了。

File: deploy/helm/templates/polaris-console-config.yaml (hunk 3 above)
Comment (zhangzerui20) [reply deploy/helm/templates/polaris-console-config.yaml:24]: 后面有需求再改吧，现在会自动部署一个 prometheus 。

**Review by chuntaojun — APPROVED** [review]

**Pull request merged**